Assets/MRTK/SDK/Experimental/MixedRealityKeyboard/MRTKTMPInputField.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.EventSystems;
+using TMPro;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// A derived class of TMP's InputField to workaround with some issues of typing on HoloLens 2
+    /// No longer used in Unity 2019.3 and later versions.
```

Comment:
I wonder if we want a custom inspector or something that includes a warning pane when used in 2019.3 or later? Or a `Debug.Log`? Something in the editor to let the developer know that this class isn't needed anymore

Reply:
I actually worked very hard to try to not introduce any custom inspector script as it will lead to me adding another asmdef and slow us down further. Technically the developers can just "don't care" about it and the class will just work for them as a regular TMP input field even in Unity 2019.3 and later. Background: we will be recommending devs to use the Unity UI/TMP input field prefabs and this script is on one of the prefabs.

Reply:
Ahh good call on the asmdef. Sounds good to me! We should just make sure it's documented well when you do / don't need this.